src/libs/UserUtils.ts

```diff
@@ -105,8 +105,8 @@ function getDefaultAvatar(accountID = -1, avatarURL?: string): React.FC<SvgProps
     // But the avatar link still corresponds to the original ID-generated link. So we extract the SVG image number from the backend's link instead of using the user ID directly
     let accountIDHashBucket: AvatarRange;
     if (avatarURL) {
-        const match = avatarURL.match(/(?<=default-avatar_)\d+(?=\.)/);
-        const lastDigit = match && parseInt(match[0], 10);
+        const match = avatarURL.match(/(default-avatar_)(\d+)(?=\.)/);
```

Comment:
can we add unit test? I think this can be easily broken in the future

Reply:
Good point, I agree. Would you mind doing that @rojiphil?

Reply:
Sure. Let me check.

Reply:
Added unit test